class/defaults.yml

```
@@ -2,3 +2,112 @@ parameters:
   topolvm:
     =_metadata: {}
     namespace: syn-topolvm
+
+    deviceclasses:
+      - name: ssd
+        volumegroup: vgssd
+        sparegb: 1
```

Lists are generally undesirable (especially when we don't have to pass them to Helm as a list).

As discussed in https://github.com/appuio/component-openshift4-logging/pull/14 I would structure this as

```suggestion
    deviceclasses:
      ssd:
        volumegroup: vgssd
        sparegb: 1
        default: true
```